src/main/java/org/ethereum/beacon/discovery/network/NettyDiscoveryServerImpl.java

```diff
@@ -57,6 +60,7 @@ public NettyDiscoveryServerImpl(InetSocketAddress listenAddress) {
               @Override
               public void initChannel(NioDatagramChannel ch) {
                 ch.pipeline()
+                    .addFirst(new ChannelTrafficShapingHandler(0, trafficReadLimit))
                     .addFirst(new LoggingHandler(LogLevel.TRACE))
```

Comment:
We're now calling `addFirst` twice so I believe this setup means the `LoggingHandler` winds up added prior to the traffic shaping handler which probably isn't what we want.

Reply:
Fixed.